tasks/test-project/test-project

```diff
@@ -164,6 +196,29 @@ const globalTasks = () =>
           )
         },
       },
+      {
+        title: 'Updating ports in redwood.toml...',
+        task: () => {
+          const REDWOOD_TOML_PATH = path.join(
+            OUTPUT_PROJECT_PATH,
+            'redwood.toml'
+          )
+          const redwoodToml = fs.readFileSync(REDWOOD_TOML_PATH).toString()
+          let newRedwoodToml = redwoodToml
+
+          newRedwoodToml = newRedwoodToml.replace(
+            /\port = 8910/,
+            'port = "${WEB_DEV_PORT:8910}"'
+          )
+
+          newRedwoodToml = newRedwoodToml.replace(
+            /\port = 8911/,
+            'port = "${API_DEV_PORT:8911}"'
+          )
+
+          fs.writeFileSync(REDWOOD_TOML_PATH, newRedwoodToml)
+        },
+      },
```

Comment:
Could this go in the Playwright fixtures instead of here, where it will be included in every built Test-project (for both CI and local dev)?

Reply:
Ummmm in theory... yes I could put it in playwright, but then I'd have to check if the toml is already modified. 
However, this change, as you can see has the fallback to the usual 8910 and 8911. So normal test projects continue to work, even without the env vars defined

Reply:
> but then I'd have to check if the toml is already modified

Not sure what this means as the toml will be know 'cause it's coming from the fixture?

Not a dealbreaker. Just feels like a weird artifact for all the local dev usage. If you decide it's best to leave here, then maybe it deserves a comment to explain why it's there in the test project?

Reply:
I've added a comment explaining why. I think given that this change has no impact unless you're actually setting the env var, I prefer to keep it in the test project generator, mainly so that the change is applied once.

Playwright fixtures can run multiple times, on different workers, which is why we'd have to add a check to see if the toml is already modified, which feels a little unnecessary to me.